### ISSUE TITLE
Define stylelint default config like @nuxtjs/eslint-config and use it

### ISSUE THREAD

**Issue by munierujp** [nuxt/create-nuxt-app#388]
## My idea
`index.js` of @nuxtjs/stylelint-config:

```js
module.exports = {
  extends: [
    'stylelint-config-standard'
  ],
  rules: [
    // TODO: Define other rules which are compatible with Nuxt
  ]
}
```

`template/_stylelint.config.js` of create-nuxt-app

```js
module.exports = {
  extends: [
    '@nuxtjs/stylelint-config'
  ],
  // add your custom rules here
  // https://stylelint.io/user-guide/configuration
  rules: []
}
```

**Comment by kiri2lov** [comment]
npm install stylelint-config-standard --save-dev

then add following line to stylelint.config.js

"extends": "stylelint-config-standard"

**Comment by clarkdo** [comment]
cc @ricardogobbosouza

**Comment by ricardogobbosouza** [comment]
I agree that we should have **@nuxtjs/stylelint-config** however simple it may be!
What do you think @clarkdo? Could we have this repository at https://github.com/nuxt/stylelint-config?

**Comment by clarkdo** [comment]
Can we add a package in https://github.com/nuxt/eslint-config/tree/master/packages ? If it's necessary, we can even rename the repo to `lint-config`

**Comment by ricardogobbosouza** [comment]
Much better :smile: 
I can create a PR added stylelint-config

**Comment by ricardogobbosouza** [comment]
It would be nice to know the team's opinion, wouldn't it?

**Comment by clarkdo** [comment]
cc @nuxt/core-team

**Comment by pi0** [comment]
As style-lint highly depends on project and UI framework does it makes sense / is possible to create general-purpose a preset? (#466 for example) AFIK `@nuxt/eslint-config` was initially refactored out of core repo to make maintenance easier and also shared across all official and community modules. It is also usable for nuxt project as they usually have shared requirements.
